test/e2e/specs/default-config/recordedit/multi-form-input.spec.js

```diff
@@ -0,0 +1,732 @@
+        }).then((toggleBtn) => {
+          expect(toggleBtn.isPresent()).toBeTruthy("toggle btn is present")
+          done();
+        }).catch(chaisePage.catchTestError(done));
+      });
+
+      it('it should not be offered for disabled columns.', () => {
+        let input = recordEditPage.getInputForAColumn('id', 1);
+        expect(input.isEnabled()).toBeFalsy("col " + 'id' + " was not disabled.");
+        let toggleBtn = recordEditPage.getColumnMultiFormButton('id');
+        expect(toggleBtn.isPresent()).toBeFalsy("toggle btn is present");
+      });
+
+
+      it('by default only the first form should be selected.', (done) => {
+        let toggleBtn = recordEditPage.getColumnMultiFormButton('markdown_col');
+        chaisePage.clickButton(toggleBtn)
+          .then(() => recordEditPage.getInputSwitchContainer('markdown_col', 1))
+          .then((inputSwitch) => recordEditPage.getParentElement(inputSwitch))
+          .then((parentElement) => {
+            expect(parentElement.getAttribute("class")).toContain('entity-active', 'Form is not selected');
+            done();
+          })
+          .catch(chaisePage.catchTestError(done));
+      });
+
+      it('the newly cloned form should not be selected.', (done) => {
+        const inputSwitch = recordEditPage.getInputSwitchContainer('markdown_col', 3)
+        const parentElement = recordEditPage.getParentElement(inputSwitch);
+        chaisePage.clickButton(cloneFormSubmitButton)
+          .then(() => {
+            expect(parentElement.getAttribute("class")).not.toContain('entity-active', 'Form is selected');
+            done();
+          })
+          .catch(chaisePage.catchTestError(done));
+      });
+
+
+      it('the form should work as a toggle when selected', (done) => {
+        const inputSwitch = recordEditPage.getInputSwitchContainer('markdown_col', 3)
+        const parentElement = recordEditPage.getParentElement(inputSwitch);
+        chaisePage.clickButton(parentElement).then(() => {
+          expect(parentElement.getAttribute("class")).toContain('entity-active', 'Form is not selected');
+        }).then(() => {
+          return chaisePage.clickButton(parentElement).then(() => {
+            expect(parentElement.getAttribute("class")).not.toContain('entity-active', 'Form is selected');
+            done();
+          }).catch(chaisePage.catchTestError(done));
+        })
+      });
+      it('previous selection should remain after closing and opening again', (done) => {
+        const inputSwitch = recordEditPage.getInputSwitchContainer('markdown_col', 3)
+        const parentElement = recordEditPage.getParentElement(inputSwitch);
+        chaisePage.clickButton(parentElement).then(() => {
+          expect(parentElement.getAttribute("class")).toContain('entity-active', 'Form is not selected');
+        }).then(() => {
+          let toggleBtn = recordEditPage.getColumnMultiFormButton('markdown_col');
+          chaisePage.clickButton(toggleBtn)
```

Comment:
Chain this promise with the previous ones instead of creating a new promise chain.